app/src/main.cpp

```diff
@@ -83,6 +101,44 @@ void loop() {
 
   monitorAudioEngine();
 
+  // Run display controls
+  disp.displayVol();
```

Comment:
The volume is already checked and adjusted in the `adjustVolume` button. Could we move this call into that function, and just display the new volume instead of checking it again?